core/src/test/scala/unit/kafka/coordinator/group/GroupMetadataTest.scala

```diff
@@ -160,6 +160,7 @@ class GroupMetadataTest {
     assertThrows(classOf[IllegalStateException], () => group.transitionTo(CompletingRebalance))
   }
 
+  @Test
```

Comment:
Nice catch! Do you mind doing a separate PR for this one? This way we can keep changes in this PR scoped in the new module.

Reply:
created https://github.com/apache/kafka/pull/13694